packages/app-builder/src/components/HelpCenter.tsx

```diff
@@ -0,0 +1,394 @@
+        },
+        {
+          label: 'Delete a List',
+          href: 'https://docs.checkmarble.com/docs/manage-rules#delete-a-list',
+        },
+        {
+          label: 'List in Scenario Builder',
+          tags: ['Formula', 'List operators'],
+          href: 'https://docs.checkmarble.com/docs/lists-in-scenario-builder',
+        },
+      ],
+      [t('navigation:workflows')]: [
+        {
+          label: 'Manage Workflows',
+          tags: ['Create', 'Update', 'Delete'],
+          href: 'https://docs.checkmarble.com/docs/manage-lists-copy',
+        },
+        {
+          label: 'Add a Decision to an Existing Case',
+          href: 'https://docs.checkmarble.com/docs/adding-a-decision-to-an-existing-case',
+        },
+      ],
+      [t('navigation:data')]: [
+        {
+          label: 'Create a Table',
+          href: 'https://docs.checkmarble.com/docs/create-a-table-or-entity',
+        },
+        {
+          label: 'Create a Field',
+          href: 'https://docs.checkmarble.com/docs/create-fields-in-a-table',
+        },
+        {
+          label: 'Edit a Field',
+          href: 'https://docs.checkmarble.com/docs/editing-a-field-in-a-table',
+        },
+        {
+          label: 'Create a link',
+          href: 'https://docs.checkmarble.com/docs/create-links-between-tables',
+        },
+        {
+          label: 'Pivot values',
+          tags: ['Workflow', 'Decision'],
+          href: 'https://docs.checkmarble.com/docs/pivot-values-1',
+        },
+        {
+          label: 'Ingesting Data',
+          tags: ['Manual', 'CSV'],
+          href: 'https://docs.checkmarble.com/docs/ingesting-data',
+        },
+      ],
+    }),
+    [t],
+  );
+
+  return { defaultTab, resources };
+}
+
+export function useTransfercheckResources() {
+  const { t } = useTranslation(['common', 'navigation']);
+
+  const location = useLocation();
+  const defaultTab = React.useMemo(() => {
+    if (location.pathname.startsWith(getRoute('/transfercheck/transfers/')))
+      return t('navigation:transfercheck.transfers');
+
+    return t('navigation:scenarios');
+  }, [location.pathname, t]);
+
+  const resources = React.useMemo(
+    () => ({
+      [t('navigation:transfercheck.transfers')]: [
+        {
+          label: 'Transfer Scoring Workflow',
+          tags: ['Score', 'Feedback', 'Transfer'],
+          href: 'https://docs.checkmarble.com/reference/transfer-scoring-workflow#send-a-transfer-and-request-a-score',
+        },
+        {
+          label: 'Transfer Data Format',
+          tags: ['Transfer', 'Data'],
+          href: 'https://docs.checkmarble.com/reference/transfer-scoring-workflow#data-format',
+        },
+        {
+          label: 'IBAN hasing',
+          tags: ['IBAN', 'Data'],
+          href: 'https://docs.checkmarble.com/reference/iban-hashing',
+        },
+      ],
+    }),
+    [t],
+  );
+
+  return { defaultTab, resources };
+}
```

Comment:
This is the content part : ATM, everything is hardcoded and not localised. I think it's unecessary ATM since our doc is in English only.

What could be done : move `href` def to the newly created file (at least links we also want to reference inside the app)